mobile-app/app/screens/AppNavigator/screens/Loans/components/CollateralizationRatioDisplayV2.tsx

```diff
@@ -37,6 +39,14 @@ export function CollateralizationRatioDisplayV2(
     totalLoanAmount: new BigNumber(props.totalLoanAmount),
   });
 
+  const isReadyForLoan =
+    props.collateralAmounts.length > 0 || props.isReadyForLoan;
+  const collateralizationRatioNum = new BigNumber(props.collateralizationRatio);
+  const hasInvalidColRatio =
+    collateralizationRatioNum.isLessThanOrEqualTo(0) ||
```

Comment:
when a user takes a huge amount of loan, the new col ratio will become zero, but we shouldn't show `Ready`
```suggestion
    collateralizationRatioNum.isLess(0) ||
```